src/box/sql/sqlInt.h

```diff
@@ -220,7 +220,6 @@ typedef sql_uint64 sql_uint64;
 
 typedef struct sql_context sql_context;
 typedef struct sql sql;
-typedef struct Mem sql_value;
```

Comment:
Please add to the commit message that you are deleting both `struct sql_value` and `sql_value`.

Reply:
Updated commit message.